### PR TITLE
[IMP] l10n_au: update documentation

### PR DESCRIPTION
**Pull request by vin-odoo**
Provides an up-to-date documentation with for
Odoo 17 with all the new and upcoming features
in this version.

### CODE REVIEW

**Comment by vin-odoo** [comment]
Hi, @odoo/localizations-doc-review would you mind checking this, please?
It is a pretty important for the Australian office :smile: 

Thanks a lot!

PS: and please see the message above :pray:

**Comment by toaa-odoo** [comment]
> Hi, @odoo/localizations-doc-review would you mind checking this, please? It is a pretty important for the Australian office 😄
> 
> Thanks a lot!
> 
> PS: and please see the message above 🙏

Hi @vin-odoo ! Working on it. Please have a look at the guidelines (https://www.odoo.com/documentation/16.0/contributing/documentation/rst_guidelines.html - 100th character limit, :guilabel:, etc...) for next time :)

As for the payroll section, it's part of the localization and therefore will stay in the localization section.

**Comment by vin-odoo** [comment]
@toaa-odoo Hi, thank you very much!
I will be more careful next time and forward that to the person who wrote the documentation, I just helped to make the PR and clean a few issues but I should have checked too :+1: 

Got it, thank you for the information.

**Review by toaa-odoo — APPROVED** [review]
@jcs-odoo @xpl-odoo Since this seems to be an urgent PR and you are both off, we are going to bypass the final review but this needs a second pass.

@auva-odoo could you r+ it when checks are ready? Forward to master.

File: static/js/reconciliation.js
Comment (AntoineVDV) [inline static/js/reconciliation.js:30]: This will conflict with the forward-port of https://github.com/odoo/documentation/pull/6896

**Comment by toaa-odoo** [comment]
@vin-odoo I cleaned up the screenshots (we don't do red squares and arrows on screenshots). I noticed a lot of screenshots were not used in the documentation but were in the PR, are they there for future use?

**Comment by vin-odoo** [comment]
@toaa-odoo Let me ask our Australian colleagues and come back to you, most likely tomorrow.

**Comment by vin-odoo** [comment]
@toaa-odoo Hi! They said that unused screenshot can be removed, thank you :pray:

**Comment by vin-odoo** [comment]
@toaa-odoo Good morning :smile: 
Do you have any ETA on when this doc could be ready? Thanks a lot!

**Comment by toaa-odoo** [comment]
> @toaa-odoo Good morning 😄 Do you have any ETA on when this doc could be ready? Thanks a lot!

Should be r+ today

File: content/applications/finance/fiscal_localizations/australia.rst
Comment (jcs-odoo) [inline content/applications/finance/fiscal_localizations/australia.rst:572]: isolated nubered list (should be #.). it seems like it follows as a bullet list afterwards.

File: content/applications/finance/fiscal_localizations/australia.rst
Comment (jcs-odoo) [inline content/applications/finance/fiscal_localizations/australia.rst:689]:
```suggestion
#. Give the batch a name. (E.g., `2024 – Weekly W1`)
```

File: content/applications/finance/fiscal_localizations/australia.rst
Comment (jcs-odoo) [inline content/applications/finance/fiscal_localizations/australia.rst:690]:
```suggestion
#. Choose the pay run's start and end date. (E.g., 01/01/2024 to 07/01/2024)
```

File: content/applications/finance/fiscal_localizations/australia.rst
Comment (jcs-odoo) [inline content/applications/finance/fiscal_localizations/australia.rst:837]:
```suggestion
   We recommend creating a batch on the fly for that payslip, in which the ETP is added as a second
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
Hi @toaa-odoo and @vin-odoo 

I'm back :)
It's quite a bold statement to say it will be merged today, Tom :D
I understand that this doc needs to be online asap, and that it is best to merge first and improve later. Therefore, I have not reviewed the content itself at all. There are still multiple points that need to be addressed before merging this PR. Some points below are comments for a later improvement, and not required for the merge. 

- In general, it's too bad the content is re-explained fully, instead of mentioning and listing the relevant existing docs in a seealso section, and only explaining the specificities of the localization. Most localization pages tend to re-explain everything from scratch, and it greatly increases the maintenance cost.
  - For example [trusted bank accounts](https://www.odoo.com/documentation/master/html/applications/finance/accounting/payments/trusted_accounts.html), Peppol, statements,  etc.
- There are some grammar mistakes, words not written in US English, or some words that were left instead of deleting them. These mistakes are quickly found and fixed with Grammarly. Always do this check please. 
- improve commit message and PR title
- 100 char length isn't well respected. It's not what's blocking the merge right now, though.
- See also [documentation pages links](https://www.odoo.com/documentation/master/contributing/documentation/rst_cheat_sheet.html#documentation-pages-hyperlinks) are not correct. 
- [Media files](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html#media-files). They're currently
  - filenames must be relevant to the content (no numbers!)
  - filenames must be written in lower-case letters
  - most images haven't been compressed with pngquant yet
  - some look small and could have used a resizing of the window's width before taking the screenshots
- Please revert the modifications done to the employment hero section: they are not specific to the version targeted by this PR (17.0) while these changes should also be applied from 15.0. These changes should be in a separate commit and PR.

Good luck :)

**Comment by vin-odoo** [comment]
@jcs-odoo Just one thing about the English part, it was on purpose that it is written in Australian English as this is targeting them. Unless we want to have fun translating the doc from us English to au English I guess, but I'm not sure if we want to bother with that.

It's quite important for the local market to have it in Australian English as they usually ask for it

**Comment by jcs-odoo** [comment]
> @jcs-odoo Just one thing about the English part, it was on purpose that it is written in Australian English as this is targeting them. Unless we want to have fun translating the doc from us English to au English I guess, but I'm not sure if we want to bother with that.
> 
> It's quite important for the local market to have it in Australian English as they usually ask for it

Oh of course, good point :)

File: content/applications/finance/fiscal_localizations/australia.rst
Comment (toaa-odoo) [reply content/applications/finance/fiscal_localizations/australia.rst:469]: I decided to leave it as is, since I'm not sure this is an Australian thing or not

**Comment by toaa-odoo** [comment]
@jcs-odoo @vin-odoo I applied the requested changes:

- Resized some of the pictures and compressed all of them (when necessary), and renamed them
- Corrected grammar mistakes etc., hopefully didn't change any Australian meaning to US
- Fixed documentation links and hyperlinks
- Reverted back the original content for Hero, I initially corrected some grammar but stopped since I believe this will be changed very soon :) (Rebranding of the app)

I'm not sure what is wrong the with PR title and commit message. AFAIK URLs can go beyond the 100th character lien

**Comment by vin-odoo** [comment]
@toaa-odoo Hello, I received some feedbacks from our friends in Australia and want to share them with you.

- the warning about not being compliant with STP should more likely be on top of the payroll section and not of the whole thing since it only concerns payroll. It also could be a bit less negative, as becoming compliant is something that is being worked on.
E.g. `"Odoo is not currently compliant with STP Phase 2. Our documentation will be updated as soon as the compliance process with the ATO has been completed.`
- Some words are written using Z and not S like it is in Australian English. (synchronize instead of synchronise)

Thank you for all the work on this documentation :pray:

**Comment by toaa-odoo** [comment]
> @toaa-odoo Hello, I received some feedbacks from our friends in Australia and want to share them with you.
> 
> * the warning about not being compliant with STP should more likely be on top of the payroll section and not of the whole thing since it only concerns payroll. It also could be a bit less negative, as becoming compliant is something that is being worked on.
>   E.g. `"Odoo is not currently compliant with STP Phase 2. Our documentation will be updated as soon as the compliance process with the ATO has been completed.`
> * Some words are written using Z and not S like it is in Australian English. (synchronize instead of synchronise)
> 
> Thank you for all the work on this documentation 🙏

Thanks for the recommendation, I made the changes. Regarding the future support of STP Phase 2, we do not mention future support and so on in documentation as nothing is ever certain, it's either there or it's not. :)

**Review by jcs-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**
